cmd/mountlib/mount.go

```diff
@@ -174,6 +176,32 @@ will see all files and folders immediately in this mode.
 				}
 			}
 
+			// Windows is not supported for background tasks
```

Comment:
In order to fix the compile problems (and it will be neater for windows) you need to factor this bit of code into a separate file and mark it with build tags for the OSes the daemon package will compile for.  This should export a single function background() say, which runs in background mode.

Then make a file which compiles for the opposite of the first file and that can just log that it isn't supported.

Does that make sense?

Check out the docs on [build tags](https://golang.org/pkg/go/build/#hdr-Build_Constraint) and see the example [syslog.go](https://github.com/ncw/rclone/blob/master/fs/log/syslog.go) and [syslog_unix.go](https://github.com/ncw/rclone/blob/master/fs/log/syslog_unix.go)

Reply:
Got it!